### PR TITLE
Checkboxes in the inspector UI should be centered vertically

### LINKED ISSUE #84: The checkboxes in the UI are misaligned
**Actual**:

**Expected**:

### PR DESCRIPTION
**Pull request by oleq**
Fix: Checkboxes in the inspector UI should be centered vertically. Closes #84.

### CODE REVIEW

**Comment by oleq** [comment]
I also hijacked this PR to clean up the tree config bars (order of components, separators, saved some space).

**Before**:



**After**:

File: src/components/pane.css [hunk 1]
```diff
@@ -41,9 +41,27 @@
...
 			}
 
-			& .ck-inspector-tree__config label {
-				margin-left: 1em;
-				margin-right: 1em;
+			& .ck-inspector-tree__config {
+				& + .ck-inspector-tree__config {
+					&::before {
+						content: "";
+						border-right: 1px solid #999;
```
Comment (jodator) [inline src/components/pane.css:48]: Wouldn't you like it to have some `var( )` definition use for `#999` config section border color?

**Comment by jodator** [comment]
Other than the question above LGTM :+1: .

File: src/components/pane.css (hunk 1 above)
Comment (oleq) [reply src/components/pane.css]: You're absolute right. Nice catch.   
  
Fixed in 36007bc.

**Pull request merged**